geoserver/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "biohub-geoserver",
+  "version": "0.0.0",
+  "description": "GeoServer for BioHub",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bcgov/biohubbc-platform.git"
+  },
+  "engines": {
+    "node": ">= 14.0.0",
+    "npm": ">= 6.0.0"
+  },
+  "scripts": {
+    "lint": "eslint src/ --ext .js,.ts",
+    "lint-fix": "npm run lint -- --fix",
+    "format": "prettier --check \"./src/**/*.{js,jsx,ts,tsx,css,scss}\"",
+    "format-fix": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,json,css,scss}\""
+  },
+  "dependencies": {
+    "typescript": "~4.1.6"
```

Comment:
Should this be a dev dependency?

Reply:
we could test and see if its okay with moving, but I'm not confident either way.  Probably an issue for another ticket